spec/features/users_can_edit_user_info_spec.rb

```diff
@@ -3,31 +3,31 @@
 feature "User can edit User info" do
   before do
     user = User.create(first_name: "Jane",
-                       last_name:  "Doe",
-                       username:   "Jane's Shop",
-                       email:      "jane@doe.com",
-                       password:   "password")
+    last_name:  "Doe",
+    username:   "Jane's Shop",
+    email:      "jane@doe.com",
+    password:   "password")
 
     user.addresses.create(type_of:   0,
-                          address_1: "1313 Mockingbird Ln",
-                          address_2: "Ste 13",
-                          city:      "Walla Walla",
-                          state:     "PA",
-                          zip_code:  "13131")
+    address_1: "1313 Mockingbird Ln",
+    address_2: "Ste 13",
+    city:      "Walla Walla",
+    state:     "PA",
+    zip_code:  "13131")
 
     user.addresses.create(type_of:   1,
-                          address_1: "123 Sesame St",
-                          address_2: "Apt 123",
-                          city:      "New York",
-                          state:     "NY",
-                          zip_code:  "12345")
+    address_1: "123 Sesame St",
+    address_2: "Apt 123",
+    city:      "New York",
+    state:     "NY",
+    zip_code:  "12345")
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Unnecessary spacing detected.